gandicli.man.rst

```diff
@@ -56,6 +56,7 @@ Namespaces:
 *  certificate change-dcv  Change the DCV for a pending certificate.
 *  certificate create      Create a new certificate.
```

Comment:
You need to update the man page for the new parameters for create command and export command